src/vsr/replica.zig

```diff
@@ -6508,6 +6514,19 @@ pub fn ReplicaType(
                 return .busy;
             }
 
+            const view_headers_op_max = self.view_headers.array.get(0).op;
+            if (view_headers_op_max > self.op) {
+                assert(view_headers_op_max <= self.op_prepare_max());
+                self.op = view_headers_op_max;
```

Comment:
Let's call `set_op_and_commit_max` here, so that its easier to find places where we update op, and also to have a trace in the log where this update happens.